web/libs/editor/src/components/BottomBar/Controls.tsx

```diff
@@ -80,7 +132,7 @@ export const Controls = controlsInjector(
       ],
     );
 
-    const RejectButton = useMemo(() => {
+    const RejectButton = memo(({ disabled, store }: { disabled: boolean, store: MSTStore }) => {
```

Comment:
Only functions created at each render, which is usual sacrifice nowadays. These two buttons don't have external dependencies, but others do, so for now I'd leave them as is to be more bulletproof, but I have a draft of new Controls which will improve this situation of course

Reply:
It's not about functions, it's about memo.
https://codesandbox.io/p/sandbox/react-memo-example-forked-f8yjgc?workspaceId=91ee8eb7-598d-4294-8b94-c2ca887b1c01

Reply:
I agree with what @Gondragos mentioned here. The `memo()` is for component declaration, as it memoizes it based on the props, such that the usage of it would not re-render unless the props had reason to change. But because this is defined not in the module space, but within another component, it will be the wrong layer of memoization. Because it is within another component, it would have to use a hook `useMemo` to memoize the function declaration, not the return value of calling the function itself. Or of course if there are no scope dependencies the best way is to just lift this out to a module level component, which you can decide at that level if `memo()` is still needed or not.